.travis.yml

```diff
@@ -88,8 +90,10 @@ script:
     # create file owned by root but writable by anyone (used by test_copy_file)
     - sudo touch /tmp/file_to_overwrite_for_easybuild_test_copy_file.txt
     - sudo chmod o+w /tmp/file_to_overwrite_for_easybuild_test_copy_file.txt
-    # run test suite
-    - python -O -m test.framework.suite 2>&1 | tee test_framework_suite.log
+    # run coverage on all travis builds except for Python 2.6
+    - if [ "x$TRAVIS_PYTHON_VERSION" != 'x2.6' ]; then coverage run -m test.framework.suite; coverage report -m --ignore-errors; fi  
```

Comment:
@shahzebsiddiqui You should include the `2>&1 | tee test_framework_suite.log` part after `coverage run -m test.framework.suite`, that's important w.r.t. a check below where we make sure there's no unexpected output from the test suite.

Reply:
@boegel Looks good to me, everything is green